### ISSUE TITLE
Feature Request: Print a single card with information

### ISSUE THREAD

**Issue by chrisHRD** [nextcloud/deck#4146]
### Description of Feature Request
When a single card is selected for editing, a option could exist in the "three dots" menu to print the card (or export to formatted PDF for printing). The printed card would contain the basic information such as:
* Title
* Tags
* Date
* Assignee (spelled out)
* Name of connected project/conversation
* Description
* Name of Attachments (with file extension) 
* Comments

### Purpose of Feature
To enable physical copies of cards to be made for situations where digital-only project management falls short.

**Comment by putt1ck** [comment]
Related to https://github.com/nextcloud/deck/issues/3250

**Comment by thefiredragon** [comment]
This would be really nice, perhaps also an option to export a pdf which could be saved directly to an folder inside nextcloud.

**Comment by calidancer** [comment]
This would be a nice addition

**Comment by UncleAndy123** [comment]
Yeah, I also want this. Would be an amazing method of sharing the information on a specific deck card.
